src/api/swg-client-event-manager-api.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2018 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AnalyticsEvent,EventOriginator} from '../proto/api_messages';
+
+/** @enum {number}  */
+export const FilterResult = {
+  CONTINUE_EXECUTING: 0,
+  STOP_EXECUTING: 1,
+};
+
+/**
+ * Defines a client event in SwG
+ * Properties:
+ * - eventType: Required. The AnalyticsEvent type that occurred.
+ * - eventOriginator: Required.  The codebase that initiated the event.
+ * - additionalParameters: Optional.  A JSON object to store generic data.
+ * - isFromUserAction: Optional.  True if the user took an action to generate
+ *   the event.
+ *
+ *  @typedef {{
+ *    eventType: (!AnalyticsEvent),
+ *    eventOriginator: (!EventOriginator),
+ *    additionalParameters: (?Object),
```

Comment:
can we reverse the order line 36/37

Reply:
Sorry i meant the additional parameters should be the last in the list of fields